src/modules/universal-discovery/components/create/create.choose.language.component.js

```diff
@@ -11,31 +11,32 @@ export default class ChooseLanguageComponent extends Component {
         this.renderOption = this.renderOption.bind(this);
 
         this.state = {
-            selectedLanguage: props.languages[0]
+            selectedLanguage: props.languages.map[props.languages.priority[0]]
```

Comment:
`.map` it's very confusing for JS developers.

Reply:
What about `props.languages.this_is_not_a_function_map[props.languages.priority[0]]`? :P

Reply:
eqeqeq